backends/openvino/README.md

```diff
@@ -18,14 +20,14 @@ executorch
 │   └── openvino
 │       ├── runtime
 │           ├── OpenvinoBackend.cpp
-│           └── OpenvinoBackend.hpp
+│           └── OpenvinoBackend.h
 │       ├── scripts
 │           └── openvino_build.sh
 │       ├── tests
 │       ├── CMakeLists.txt
 │       ├── README.md
 │       ├── __init__.py
-│       ├── openvino_functions.yaml
+│       ├── unsupported_openvino_functions.yaml
```

Comment:
With the new change, I believe we dont need this yaml file. Is this correct?

Reply:
Forgot to update the readme file. Changed now.